scopes/mdx/ui/create-heading/create-heading.module.scss

```diff
@@ -1,13 +1,23 @@
 .mdxLinkedHeading {
-  padding-top: 40px;
-  border-bottom: 1px solid #ededed;
-  padding-bottom: 8px;
+  margin-top: 40px;
   width: 100%;
+  // lets the user decide how he wants to align the text
```

Comment:
I think you can leave a more detailed comment:
```
mdx is aligning content using the deprecated html `align` attribute.
this polyfills this behavior.
```

Reply:
also, consider using `text-align`

Reply:
I cant use `text-align` because it already uses flex. so I just used it like it is